src/main/resources/camunda/claimant_response_spec.bpmn

```diff
@@ -302,10 +302,10 @@
           <camunda:inputParameter name="caseEvent">NOTIFY_RESPONDENT1_FOR_CLAIMANT_AGREED_REPAYMENT</camunda:inputParameter>
         </camunda:inputOutput>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_ADMIT_PART_ACCEPT_REPAYMENT</bpmn:incoming>
+      <bpmn:incoming>Flow_0qiuirk</bpmn:incoming>
```

Comment:
perhaps needs a better naming of the flow?

Reply:
As discussed, there should be no change on that part